### PR TITLE
Baseline System 11 RMD

### PR DESCRIPTION
**Pull request by jugonzal07**
first cut at system 11 baseline

### CODE REVIEW

File: rct229/ruletest_engine/ruletest_jsons/system_types/System_11.1_VAV_SZ.json [hunk 1]
```diff
@@ -0,0 +1,99 @@
...
+                                {
+                                    "id": "System Type 11.1",
+                                    "cooling_system": {
+                                        "id": "CHW Coil 1",
+                                        "cooling_system_type": "FLUID_LOOP",
+                                        "chilled_water_loop": "Chiller Loop 1"
+                                    },
+                                    "heating_system": {
+                                        "id": "HHW Coil 1",
+                                        "heating_system_type": "FLUID_LOOP",
```
Comment (yunjoonjung-PNNL) [inline rct229/ruletest_engine/ruletest_jsons/system_types/System_11.1_VAV_SZ.json:38]: Could you check if this needs to be ```ELECTRIC_RESISTANCE``` according to [HERE](https://github.com/pnnl/ruleset-checking-tool/blob/develop/docs/ruleset_functions/baseline_systems/baseline_hvac_sub_functions/is_hvac_sys_heating_type_elec_resistance.md#:~:text=heating_system_b.heating_system_type%20%3D%3D%20%22ELECTRIC_RESISTANCE%22)?

**Review by yunjoonjung-PNNL — APPROVED** [review]
Approved with one comment for a double-check purpose.

File: rct229/ruletest_engine/ruletest_jsons/system_types/System_11.1_VAV_SZ.json (hunk 1 above)
Comment (jugonzal07) [reply rct229/ruletest_engine/ruletest_jsons/system_types/System_11.1_VAV_SZ.json]: I believe this has been updated. I hope so!

**Comment by weilixu** [comment]
Updated to 0.23 and add the test jsons to the test files

**Pull request merged**